contents/blog/the-posthog-array-1-33-0.md

```diff
@@ -0,0 +1,124 @@
+---
+date: 2022-02-28
+title: Array 1.33.0
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Release notes", "Product updates"]
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+excerpt: Introducing a full multivariate Experimentation suite, granular permissions on dashboards, Altinity Cloud support for ClickHouse, configuring your instance from the PostHog UI and 290+ improvements/fixes.
+---
+
+PostHog 1.33.0 introduces a full Experimentation suite! A/B/C/... changes to your product to ensure you maximize value for your end users. For Enterprise customers, you can now set granular permissions on dashboards. An easier way to configure your instance, support for Altinity Cloud, insight legends, insight permalinks and 290+ improvements/fixes are included.
```

Comment:
```suggestion
PostHog 1.33.0 introduces a full Experimentation suite, enabling you to run multivariate experiments to ensure you maximize value for your end users. 

For Enterprise customers, we've also added the ability to set granular permissions on dashboards, support for Altinity Cloud, insight legends and insight permalinks. Plus, there are over 290+ improvements and bugfixes included!
```

@paolodamico can you double check this? I've updated, but I'm still unclear on if Altinity cloud, legends and permalinks are enterprise features.